packages/jest-matchers/src/assertMatchers.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+ /* eslint-disable max-len */
+
+'use strict';
+
+import type {
+  MatchersObject,
+} from 'types/Matchers';
+
+const GLOBAL_STATE = Symbol.for('$$jest-matchers-object');
+
+const expectMatchers: MatchersObject = {
+  assertions(expected: number) {
+    global[GLOBAL_STATE].state.expectedCounts.total = expected;
```

Comment:
can we move this into `index.js` and directly assign as
```js
expect.assertions = ...
```

accessing the state through a symbol can get out of hand really quickly